music_assistant/server/controllers/player_queues.py

```diff
@@ -631,10 +632,26 @@ async def player_ready_for_next_track(
         queue = self.get_active_queue(queue_or_player_id)
         cur_index = self.index_by_id(queue.queue_id, current_item_id)
         cur_item = self.get_item(queue.queue_id, cur_index)
-        next_index = self.get_next_index(queue.queue_id, cur_index)
-        next_item = self.get_item(queue.queue_id, next_index)
-        if not cur_item or not next_item:
-            raise QueueEmpty("No more tracks left in the queue.")
+        idx = 0
```

Comment:
whats idx doing here ?

Reply:
> whats idx doing here ?

Good point. Forgot to add the increment, which will try to fetch the 'next next' item if the 'next' item is unplayable.